src/Employer/Employer.Web/ViewModels/ApplicationReview/ApplicationReviewViewModel.cs

```diff
@@ -64,6 +64,8 @@ public class ApplicationReviewViewModel : ApplicationReviewRouteModel
         public bool IsApplicationShared => Status == ApplicationReviewStatus.Shared;
         public bool IsApplicationEmployerInterviewing => Status == ApplicationReviewStatus.EmployerInterviewing;
         public bool IsApplicationSharedByProvider => DateSharedWithEmployer.HasValue;
+        public bool CanShowRadioButtonReview => Status == ApplicationReviewStatus.New && !IsApplicationShared;
+        public bool CanShowRadioButtonInterviewing => (Status == ApplicationReviewStatus.New || Status == ApplicationReviewStatus.InReview) && !IsApplicationShared;
```

Comment:
Should have some tests around this logic 😁

Reply:
done I think, let me know if okay enough